app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    vaccinated_count = 0
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+            vaccinated_count += 1
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
+        except OutdatedVaccineError:
+            return "All friends should be vaccinated"
```

Comment:
These two errors inherit from the same class. So how can you simplify these two statements?